test/micro/org/openjdk/bench/jdk/incubator/foreign/points/support/PanamaPoint.java

```diff
@@ -43,8 +43,8 @@
 public class PanamaPoint implements AutoCloseable {
 
     public static final MemoryLayout LAYOUT = MemoryLayout.ofStruct(
-        MemoryLayouts.JAVA_INT.withOrder(ByteOrder.nativeOrder()).withName("x"),
-        MemoryLayouts.JAVA_INT.withOrder(ByteOrder.nativeOrder()).withName("y")
+        MemoryLayouts.C_INT.withOrder(ByteOrder.nativeOrder()).withName("x"),
```

Comment:
Do we need the 'withOrder' call?

Reply:
No, that's not needed. Must have been a left over from experimenting :)